nlp/README.md

```diff
@@ -20,4 +20,8 @@ In this tip we look at `pke`, an alternative to Gensim for keyword extraction.
 
 -  [_Explainable transformers using SHAP_ (22/04/2021)](2021_04_22_shap_for_huggingface_transformers):
 BERT, explain yourself! 📖
-Up until recently language model predictions have lacked transparency. In this tip we look at `SHAP`, a way to explain your latest transformer based models.
\ No newline at end of file
+Up until recently language model predictions have lacked transparency. In this tip we look at `SHAP`, a way to explain your latest transformer based models.
+
+-  [_Transformer-based Data Augmentation (18/06/2021)](2021_06_18_data_augmentation):
```

Comment:
missing ending '_' so that the title isn't italic. You can check it out here: https://github.com/ml6team/quick-tips/tree/feature/nlp_data_aug/nlp